doc/examples/visualization.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "50d3669e",
+   "metadata": {},
+   "source": [
+    "# Visualization gallery"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c01e724e-1d4e-41e7-bb76-742f354fd15f",
+   "metadata": {},
+   "source": [
+    "The visualization methods implemented in PEtab Select are demonstrated here. These methods generally visualize the output of a model selection task, so the input is generally a list of already-calibrated models.\n",
+    "\n",
+    "Here, some calibrated models that were saved to disk with `petab_select.model.models_to_yaml_list` are loaded and used as input. This is the result of a forward selection with the problem provided in `calibrated_models`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ca6ce5b4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import petab_select\n",
+    "\n",
+    "models = petab_select.models_from_yaml_list(\n",
```

Comment:
I think showing a table of that data might be helpful for newcomers to know what the visualizations are based on. Maybe just a table of model + criterion value + predecessor could be enough.

Reply:
Nice, thanks